docs/quick_start.md

```diff
@@ -59,7 +59,7 @@ configuration:
 ```
 --run-config-search-max-concurrency 2 \
 --run-config-search-max-instance-count 2 \
---run-config-search-preferred-batch-size-disable true
```

Comment:
Was this the conclusion of talks with Iman? That the value should be unspecified?  What was the behavior if 'True' is also specified here? I would expect an error

Reply:
Yes. Now, if a value is specified it will result in a CLI error. Before, you could put whatever you wanted: false, cat, rutabaga... and the end result would always be having the variable set to true.

Reply:
ok, good fix. I created a ticket to improve our CLI protection